### ISSUE TITLE
Fish values are wrong (inconsistent with each other)

### ISSUE THREAD

**Issue by Xpyder** [CleverRaven/Cataclysm-DDA#17113]
the nutrition values for preserved fish (esp salted and pickled) vs raw fish seem to be wrong (adding salt shouldn't double the calorie value of your food)

the fish nutrition values per volume are probably wrong compared with "meat", though I don't know if they should be higher or lower (per volume)

cooked fish (fish_cooked) seems to be missing in comestibles.json and I can't find it anywhere else (suspect making some will throw an error)

pickled fish charges per volume doesn't match most fish (most fish is 2 charges per volume), this would require changing lots of recipes.  If it is actually twice the size from soaking up the brine, then it should still take the same amount for regular fish in recipes.  Otherwise it should be changed to 2 charges like all other fish and have it's nutrition adjusted for consistency.

**Comment by Kadian** [comment]
Cooked fish is found in json\items\comestibles\carnivore. Directly under the fillet of fish.

As to the nutrition values: They do feel a bit odd, yes. Meat has similar issues, although not as big. 1 Chunk of Meat (28 Nut) will create 3 charges of Meat Jerky (12 each, 36 in total. ; pickled meat is even worse in this regard, as 1 Chunk of meat will produce 2 charges of pickled meat, with 45 Nut each, making 90 Nut out of 28 Nut and some 0 Nut ingredients.

**Comment by Xpyder** [comment]
Pickled meat should only produce 1 charge per meat now (there are some recent changes as of about half an hour ago).  If that's not the case on tomorrows build please post back so I can double check my work.

I rebalanced some of the preserved meats a little while ago on the assumption that cooked meat gives 50 nutrition, I think the current nutrition value is a bug.  Probably a leftover issue from a partially implemented change from "nutrition" to a "calories + vitamins" system which hasn't been completed yet.

Statistically in game, fish is supposed to 2 servings per 1 volume vs meat's 1 serving per volume, and I assume they are supposed to have similar but slightly different calorie values per volume (making the fish calorie per serving close to half that of meat since it has 2 servings).

If that assumption is wrong, and fish is supposed to be twice the nutritional value of meat per volume then things still need to be rebalanced, just in a different direction.

I'm not a nutrition expert so I don't know the real world comparison between fish and meat so I won't put in a PR myself unless someone tells me what they should be.

**Comment by Rivet-the-Zombie** [comment]
>Chunk of Meat (28 Nut)

This value is way lower than it used to be. I wonder what happened to it?

**Comment by Xpyder** [comment]
This occurred when it was changed from flat nutrition to vitamins and calories.
The in game calorie count is accurate to real life (roughly), so either there's an issue with the conversion from calories to nutrition or more likely it needs to include protein in the nutrition conversion.

Meat is important in a survival situation not just for the calories, but also for the protein which breaks down into components that your body needs to build and repair almost all it's cells, restore lost blood, and produce lots of your chemicals like enzymes and hormones.  Without protien your body has to either cannibalize from other areas of itself (which makes you weaker) or work really hard to produce them from raw materials which requires a lot of calories and other foods.

TL:DR the more defined nutrition system need to take protein into account but doesn't.

**Comment by Xpyder** [comment]
There seems to be a consensus that the new nutrition values for raw/cooked fish/meat are where they should be since they're changing Nutrition to Calories. (#17123)

The direct recipes should probably be rebalanced given these new values.

**Comment by Leland** [comment]
Closing - minimal discussion going on here, and we're approaching this being a year old. Feel free to comment here if you still want it open, or make a new issue.
